apps/dcs-admin/src/app/features/components/sidebar/sidebar.component.ts

```diff
@@ -79,7 +79,7 @@ export class SidebarComponent implements OnInit {
     if (title.charAt(0) === '#') {
       title = title.slice(1);
     }
-    title = '/' + title.split('/')[1];
+    title = '/' + title.split('/')[2];
```

Comment:
This is a lucky number anti-pattern, can you create a variable with the description of what it is and what is for? like this example
```
const PATH_NAME_POSITION = 2;
title = '/' + title.split('/')[PATH_NAME_POSITION];
```

Reply:
Done